src/read/elf/relocation.rs

```diff
@@ -736,3 +739,131 @@ impl<Endian: endian::Endian> Relr for elf::Relr64<Endian> {
         }
     }
 }
+
+/// Compact relocation
+///
+/// The specification has been submited here: <https://groups.google.com/g/generic-abi/c/ppkaxtLb0P0/m/awgqZ_1CBAAJ>.
+#[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
This warning is a hint that the code is wrong. I think you should make the fields public instead.